.github/workflows/build-deps.yml

```diff
@@ -130,6 +135,29 @@ jobs:
           -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF
         cmake --build build_arm64 --config MinSizeRel
 
+  build-python:
+    if: ${{ inputs.python_build }}
+
+    runs-on: [self-hosted, Windows, ARM64, WASM]
```

Comment:
could it be cross-compiled?

Reply:
Not yet, "cross-compiling" is just about adding parameter to update_python.py script. We just need to verify this is actualy usefull package first.